src/tests/test_integration/test_comments.py

```
@@ -0,0 +1,189 @@
+    user = UserModel(
+        email="another@email.com",
+        is_active=True,
+        _hashed_password="hashed"
+    )
+    db_session.add(user)
+    await db_session.commit()
+    await db_session.refresh(user)
+    return user
+
+@pytest.mark.asyncio
+async def test_reply_comment_and_notification(
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)